lib/index.js

```diff
@@ -7,11 +7,13 @@
 var fluentChaining = require('./rules/fluent-chaining'),
     arrayIndentation = require('./rules/array-indentation'),
     callIndentation = require('./rules/call-indentation'),
-    noMultipleInlineFunctions = require('./rules/no-multiple-inline-functions');
+    noMultipleInlineFunctions = require('./rules/no-multiple-inline-functions'),
+    noMultilineVarDeclaration = require('./rules/no-multiline-var-declaration');
 
 module.exports.rules = {
    'fluent-chaining': fluentChaining,
    'array-indentation': arrayIndentation,
    'call-indentation': callIndentation,
    'no-multiple-inline-functions': noMultipleInlineFunctions,
+   'no-multiline-var-declaration': noMultilineVarDeclaration,
```

Comment:
This plugin name is defined in https://github.com/silvermine/eslint-config-silvermine/pull/5 as `no-multiline-var-declarations` (plural). The name should either be fixed here or in that PR.